### PR TITLE
Created visual display page folder in new dir structure

### LINKED ISSUE #69: Create folder for visual display pages inside the results folder
Create folder for visual display pages inside the results folder and fix the neutron visual display page links.

### PR DESCRIPTION
**Pull request by bb511**
#### Description of Work

Fixes #69 
Visual display pages now reside in `/fitbenchmarking/fitbenchmarking/results/neutron/VDPages/` and the tables link to them correctly.

#### Testing Instructions

1. Run `example_runScripts.py` in the mantidpython console.
2. Check if the links in the tables available at `/fitbenchmarking/fitbenchmarking/results/neutron/Tables/` point to the correct VDpages

**Note:** VDpages are still broken for the time. They will be fixed in #64 .

### CODE REVIEW

**Comment by Anders-Markvardsen** [comment]
please fix merge conflict

**Comment by Anders-Markvardsen** [comment]
As discussed in a separate issue have results by default are saved to folder it the directory of the script running the fitbenchmarking

**Comment by Anders-Markvardsen** [comment]
Please change name VDPages to support_pages and put within tables folder

File: fitbenchmarking/results_output.py [hunk 1]
```diff
@@ -141,25 +139,33 @@ def build_indiv_linked_problems(results_per_test, group_name):
...
-def build_visual_display_page(prob_results, group_name):
+def build_visual_display_page(prob_results, group_name, results_dir):
     """
     Builds a page containing details of the best fit for a problem.
     @param prob_results:: the list of results for a problem
     @param group_name :: the name of the group, e.g. "nist_lower"
     """
 
+    # Directory that holds all the visual display pages
+    VDPages_dir = os.path.join(results_dir, "neutron", "VDPages")
```
Comment (Anders-Markvardsen) [inline fitbenchmarking/results_output.py]: Please change to VDPages_dir to support_pages_dir. Also please use PEP 8 style variable naming convention

File: fitbenchmarking/fitting_benchmarking.py [hunk 2]
```diff
@@ -210,7 +213,7 @@ def do_fitting_benchmark_one_problem(prob, group_results_dir, minimizers, use_er
     return results_fit_problem
 
 
-def make_plots(prob, group_results_dir, best_fit, wks, previous_name, count, user_func):
+def make_plots(prob, visuals_dir, best_fit, wks, previous_name, count, user_func):
```
Comment (Anders-Markvardsen) [inline fitbenchmarking/fitting_benchmarking.py]: Is visuals_dir the support pages dir?

File: fitbenchmarking/fitting_benchmarking.py [hunk 3]
```diff
@@ -222,9 +225,14 @@ def make_plots(prob, group_results_dir, best_fit, wks, previous_name, count, use
     @param count :: number of different starting points for one problem
     @param user_func :: fitting function
     '''
+    if "neutron" in visuals_dir:
+        VDPage_dir = os.path.join(visuals_dir, "VDPages")
```
Comment (Anders-Markvardsen) [inline fitbenchmarking/fitting_benchmarking.py]: Is this code correct?

File: fitbenchmarking/fitting_benchmarking.py (hunk 2 above)
Comment (bb511) [reply fitbenchmarking/fitting_benchmarking.py]: Yes.

File: fitbenchmarking/fitting_benchmarking.py [hunk 4]
```diff
@@ -222,9 +225,14 @@ def make_plots(prob, group_results_dir, best_fit, wks, previous_name, count, use
     @param count :: number of different starting points for one problem
     @param user_func :: fitting function
     '''
+    if "neutron" in visuals_dir:
+        VDPage_dir = os.path.join(visuals_dir, "tables", "support_pages")
```
Comment (Anders-Markvardsen) [inline fitbenchmarking/fitting_benchmarking.py]: please change VDPage_dir to support_page_dir. Please always use PEP 8 style variable names

File: fitbenchmarking/fitting_benchmarking.py (hunk 2 above)
Comment (Anders-Markvardsen) [reply fitbenchmarking/fitting_benchmarking.py]: also where you change function name I would expect this name to change in documentation. If no nothing documented for old variable please add documentation

**Comment by Anders-Markvardsen** [comment]
tested and works

**Pull request merged**